src/aap_eda/services/activation/engine/kubernetes.py

```diff
@@ -202,7 +202,9 @@ def update_logs(self, container_id: str, log_handler: LogHandler) -> None:
 
                 for line in log.splitlines():
                     timestamp, content = line.split(" ", 1)
-                    log_handler.write(content)
+                    log_handler.write(
+                        lines=content, flush=True, timestamp=False
```

Comment:
@hsong-rh I think Flush should also be False since we don't want to write every line to the DB right away we can use the auto flush which will reduce the writes to the DB.

Reply:
👌